[Docker] make install-pip-package failed

Hi,

I am trying to follow the docker example in the document page.
after ./run.sh, I am inside the container, but python3 -c "import open3d" results in: No model named 'open3d'.
So, I suppose python is not setup for open3d.
Then, I try to run "make install-pip-package" inside the build folder. 
Below is the error output:

```
root@open3d-xvfb:~/open3d/build#  make install-pip-package
[  1%] Built target EncodeShader
[  2%] Built target ShaderFileTarget
[ 26%] Built target Visualization
[ 29%] Built target tinyfiledialogs
[ 61%] Built target Core
[ 79%] Built target IO
[ 80%] Built target Open3D
[100%] Built target open3d
[100%] Built target python-package
Traceback (most recent call last):
  File "setup.py", line 108, in <module>
    zip_safe=False,
  File "/usr/local/lib/python3.6/dist-packages/setuptools/__init__.py", line 143, in setup
    return distutils.core.setup(**attrs)
  File "/usr/lib/python3.6/distutils/core.py", line 134, in setup
    ok = dist.parse_command_line()
  File "/usr/local/lib/python3.6/dist-packages/setuptools/dist.py", line 572, in parse_command_line
    result = _Distribution.parse_command_line(self)
  File "/usr/lib/python3.6/distutils/dist.py", line 472, in parse_command_line
    args = self._parse_command_opts(parser, args)
  File "/usr/local/lib/python3.6/dist-packages/setuptools/dist.py", line 887, in _parse_command_opts
    nargs = _Distribution._parse_command_opts(self, parser, args)
  File "/usr/lib/python3.6/distutils/dist.py", line 534, in _parse_command_opts
    if not issubclass(cmd_class, Command):
TypeError: issubclass() arg 1 must be a class
src/Python/CMakeFiles/pip-package.dir/build.make:57: recipe for target 'src/Python/CMakeFiles/pip-package' failed
make[3]: *** [src/Python/CMakeFiles/pip-package] Error 1
CMakeFiles/Makefile2:901: recipe for target 'src/Python/CMakeFiles/pip-package.dir/all' failed
make[2]: *** [src/Python/CMakeFiles/pip-package.dir/all] Error 2
CMakeFiles/Makefile2:940: recipe for target 'src/Python/CMakeFiles/install-pip-package.dir/rule' failed
make[1]: *** [src/Python/CMakeFiles/install-pip-package.dir/rule] Error 2
Makefile:398: recipe for target 'install-pip-package' failed
make: *** [install-pip-package] Error 2
root@open3d-xvfb:~/open3d/build# python3 -c "import open3d"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'open3d'
```

Any ideas on what might go wrong?
Thanks!

--- a/src/Python/package/setup.py
+++ b/src/Python/package/setup.py
@@ -26,14 +26,22 @@
 
 from setuptools import setup, find_packages
 import glob
+
+# Force platform specific wheel
 try:
     from wheel.bdist_wheel import bdist_wheel as _bdist_wheel
+    # https://stackoverflow.com/a/45150383/1255535
     class bdist_wheel(_bdist_wheel):
         def finalize_options(self):
             _bdist_wheel.finalize_options(self)
             self.root_is_pure = False
 except ImportError:
+    print('Warning: cannot import "wheel" package to build platform-specific wheel')
+    print('Install the "wheel" package to fix this warning')
     bdist_wheel = None
+
+cmdclass = {'bdist_wheel': bdist_wheel} if bdist_wheel is not None else dict()
+
 
 # Read requirements.txt
 with open('requirements.txt', 'r') as f:
@@ -86,7 +94,7 @@ setup(
     description=[
         "Open3D is an open-source library that supports rapid development of software that deals with 3D data."
     ],
-    cmdclass={'bdist_wheel': bdist_wheel},
+    cmdclass=cmdclass,
     install_requires=install_requires,
     include_package_data=True,
     data_files=data_files,

